provider/README.md

```diff
@@ -336,7 +336,7 @@ When multiple query parameters are specified, they should all apply to the retur
 [Top][toc]
 
 ## Realtime Data
-All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in an available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should generally reconcile with data from the status changes enpdoint.
+All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should reconcile with data from the status changes enpdoint. This data should be refreshed at most every 5 minutes.
```

Comment:
So be a little pedantic, I think "at most" in the context of time can be ambiguous. "at most every 5 minutes" could be interpreted as "no more frequently than once every 5 minutes" rather than the intended interpretation of "at least once every five minutes."

I would suggest instead:


```suggestion
All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should reconcile with data from the status changes endpoint. The data returned by this endpoint should be as close to realtime as possible, but in no case should it be more than 5 minutes out-of-date.
```

Reply:
Updated the language to reflect your suggestion @jfh01